lanesawyer/ratchet
# chore: Well-known file array

# chore: Well-known file array

## Why
- Cleaner to move all the well-known files to ignore to its own variable in the config settings

## What
- New `WELL_KNOWN_FILES` array

## How
- Moved code into an array, loop over that to decide whether or not to skip it

## Testing
- Run `turn`
- See that it doesn't run on any of the well-known files

## PR Checklist
- [x] Tests, linting, and formatting checks pass.
